src/components/PostDetails.tsx

```diff
@@ -154,7 +145,7 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
           </>
         )}
 
-        {loaded && !hasError && !visible && (
+        {!isLoading && !hasError && !visible && (
```

Comment:
consider moving long ternaries to a separate const with consistent name and use the const here